internal/controller/controllers/installenv_controller.go

```diff
@@ -44,9 +44,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+type InstallEnvConfig struct {
+	ImageType models.ImageType `envconfig:"ISO_IMAGE_TYPE" default:"minimal-iso"`
```

Comment:
just a question regarding design. 
Why iso_type is os env and not part of crd with some default value?

Reply:
We don't want to enable the users to set iso type at this point.